versions/development/SPEC.md

```diff
@@ -538,20 +534,15 @@ Below are the valid results for operators on types.  Any combination not in the
 |`Int`|`>=`|`Int`|`Boolean`||
 |`Int`|`<`|`Int`|`Boolean`||
 |`Int`|`<=`|`Int`|`Boolean`||
-|`Int`|`+`|`String`|`String`||
-|`String`|`+`|`Float`|`String`||
-|`String`|`+`|`Int`|`String`||
-|`String`|`+`|`String`|`String`||
+|`String`|`+`|`String`|`String`|Concatenation|
 |`String`|`==`|`String`|`Boolean`||
 |`String`|`!=`|`String`|`Boolean`||
 |`String`|`>`|`String`|`Boolean`||
 |`String`|`>=`|`String`|`Boolean`||
 |`String`|`<`|`String`|`Boolean`||
```

Comment:
Which sort order is this? WDL says a string is unicode. Python default sort is not the 'official' unicode sort order IIRC.
https://www.unicode.org/reports/tr10/#Introduction